pages/index.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
```

Comment:
We should put the index.html in the root of the project and all other pages can live in the pages directory